docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,82 @@
+[role="xpack"]
+[[xpack-logs-analysis-page]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+The *Analysis* page in the Logs app allows you to automatically detect some kinds of log anomalies using machine learning.
```

Comment:
I think it's worth stating right off the bat that this page isn't applicable if you don't have the appropriate ML features available. For example:
```suggestion
If the {ml} {anomaly-detect} features are enabled, you can use the 
the *Analysis* page in the Logs app to automatically detect some kinds of log anomalies.
```